src/main/groovy/org/pih/warehouse/importer/OutboundStockMovementExcelImporter.groovy

```diff
@@ -58,7 +56,8 @@ class OutboundStockMovementExcelImporter extends AbstractExcelImporter {
     }
 
     List<Map> getData() {
-        return excelImportService.convertColumnMapConfigManyRows(workbook, columnMap, null, propertyMap)
+        Holders.grailsApplication.mainContext.getBean("excelImportService")
+                .convertColumnMapConfigManyRows(workbook, columnMap, null, null, propertyMap)
```

Comment:
There is signature of `convertColumnMapConfigManyRows` method:
![image](https://github.com/openboxes/openboxes/assets/83239466/c39f35ea-fa42-45c6-8740-1615fa3a1aa4)
So, the second `null` was missing and I got error connected to being unable to find appropriate signature